tools/ci/tests/stations_test.py

```diff
@@ -0,0 +1,29 @@
+import re
+
+from unittest import TestCase
+
+from tools.ci.sim_results import SimRecordedData
+from tools.ci.common import TestOptions, Decorators
+
+
+@Decorators.artery_test
+@Decorators.defines_test_options({
+    'vehicles': None
```

Comment:
Not just vehicles, but any dynamic entity managed by the TraCI node manager, e.g. persons.
Furthermore, a scenario may contain fixed stations as well, e.g. road side units. See the `artery.StaticNodeManager`.